src/precision.hpp

```diff
@@ -107,6 +107,49 @@ inline static Precision fromString(const std::string& s) {
     return it->second;
 }
 
+inline static Precision kfsPrecisionToOvmsPrecision(const std::string& s) {
+    static std::unordered_map<std::string, Precision> precisionMap{
+        {"BOOL", Precision::BOOL},
+        {"FP64", Precision::FP64},
+        {"FP32", Precision::FP32},
+        {"FP16", Precision::FP16},
+        {"INT64", Precision::I64},
+        {"INT32", Precision::I32},
+        {"INT16", Precision::I16},
+        {"INT8", Precision::I8},
+        {"UINT64", Precision::U64},
+        {"UINT32", Precision::U32},
+        {"UINT16", Precision::U16},
+        {"UINT8", Precision::U8}};
```

Comment:
It's worthy here to add TODO for BYTES datatype
```suggestion
        {"UINT8", Precision::U8}};
        // {"BYTES", Precusion::??} // TODO decide what to do with BYTES 
```

Reply:
done